src/Sylius/Component/Core/Storage/CartStorageInterface.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Core\Storage;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+
+interface CartStorageInterface
```

Comment:
Just a thought: we can make it accept / return `OrderInterface` instances instead of cart ids so we do not have to worry about garbage being put inside this storage & we put `OrderInterface` instead of `mixed`. Then we can skip the `CartId` part from method names completely and end up with them like `getForChannel(ChannelInterface $channel): OrderInterface`.